src/features/bivariate_manager/readme.md

```diff
@@ -0,0 +1,53 @@
+## Bivariate Manager
+
+Bivariate manager is a tool for analyzing and finding new insights from a set of suggested layers.
+
+### How to activate
+
+It requires feature bivariate_manager turned on.
+Then click "Bivariate Matrix" panel to see the matrix and start working with it.
```

Comment:
Since this documentation is for developers and not users, it is worth focusing on how to activate the feature programmatically, i.e. what to call after pressing "Bivariate Matrix" button

Reply:
it's fully ui related feature, not expected to be called programmatically, as it requires resource atom data to be used.
will remove this line if it confuses 🤔